mlflow/models/evaluation/default_evaluator.py

```diff
@@ -412,6 +423,10 @@ def _do_plot(**kwargs):
     )
 
 
+def _get_metrics_values(metrics):
+    return {name: MetricValue(aggregate_results={name: value}) for name, value in metrics.items()}
```

Comment:
Do we want to rename this to `_get_aggregate_metric_values()` since this just return `aggregate` and not scores and justification. Thoughts?